discovery-infra/tests/test_e2e_install.py

```diff
@@ -4,25 +4,23 @@
 
 from tests.base_test import BaseTest
 from tests.config import ClusterConfig, TerraformConfig
-from tests.conftest import get_available_openshift_versions
+from tests.conftest import get_available_openshift_versions, get_api_client
 
 
 class TestInstall(BaseTest):
 
     @JunitTestSuite()
-    @pytest.mark.parametrize("openshift_version", get_available_openshift_versions())
+    @pytest.mark.parametrize("openshift_version", sorted(get_available_openshift_versions()))
```

Comment:
I think it's better for adding the `sorted` on the return value of `get_available_openshift_versions`
```suggestion
    def get_available_openshift_versions() -> List[str]:
        ...
        return sorted(available_versions)
```

Reply:
Ack